### PR TITLE
docs: add hint that this requires docker

### PR DESCRIPTION
**Pull request by kilsenp**
To solve issue #473.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,12 +24,16 @@ with Flow().add(uses=my_image_tag):
...
 ## Contribute to This Repository
 
-### Create a New Executor
-
+### Installation
 ```bash
 git clone https://github.com/jina-ai/jina-hub.git
 
 pip install "jina[hub]"
+```
```
Comment (JoanFM) [inline README.md:32]: I would put this in Prerequisites together with pip install jina[hub] and then keep the clone and run to create the executor.

Comment (kilsenp) [reply README.md:32]: You mean like this?

```
# Prerequisites
pip install "jina[hub]"
Install Docker.

# Create a New Executor
git clone https://github.com/jina-ai/jina-hub.git

jina hub new --type pod
```

Comment (JoanFM) [reply README.md:32]: Yes kind of

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**